recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
-{% set version = "0.6.0" %}  # PEP 386
+{% set version = "0.8.0" %}  # PEP 386
```

Comment:
it should be 0.9.0 I guess

Reply:
thanks; this does not actually affect the nightly build, it uses main branch and not a source release

Reply:
fixed